api/src/test/java/org/pmiops/workbench/api/UserMetricsControllerTest.java

```diff
@@ -52,26 +54,30 @@
   private UserMetricsController userMetricsController;
   private static final Instant NOW = Instant.now();
   private FakeClock clock = new FakeClock(NOW);
+  private long workspace_1_Id = 1l;
```

Comment:
Please don't use underscores in java variable names (exception: constants in all caps)